drydock/templates/kustomized/tutor16/base/apps/openedx/config/cms.env.yml

```diff
@@ -0,0 +1,40 @@
+SITE_NAME: "{{ CMS_HOST }}"
```

Comment:
Please include the uwsgi template from [tutor](https://github.com/overhangio/tutor/blob/master/tutor/templates/apps/openedx/uwsgi.ini)